From 19ceb5ef4711ebea49deaca990d1cc10d820970f Mon Sep 17 00:00:00 2001
From: Bryan Brattlof <bb@ti.com>
Date: Tue, 1 Feb 2022 14:37:34 -0600
Subject: [PATCH 8/8] readme: add j721s2 to the list of supported chips

k3conf now has support for the j721s2. Update the README to reflect this

Signed-off-by: Bryan Brattlof <bb@ti.com>
---
 README.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.md b/README.md
index cbb559d..ffcade3 100644
--- a/README.md
+++ b/README.md
@@ -23,7 +23,7 @@ dimension. Use at your own risk. And keep the reset button in reach.
 SUPPORT:
 --------
 
-K3CONF currently supports AM654, J721E, J7200, AM64x and AM62x Devices.
+K3CONF currently supports AM654, J721E, J7200, AM64x, AM62x and J721S2 Devices.
 Legacy OMAP and DRA7 platforms are not supported.
 
 
-- 
2.17.1

